File: csharp/beecrowd1008.cs
using System;
using System.Globalization;

class Programa
{
    static void Main(string[] args)
    {
        CultureInfo CI = CultureInfo.InvariantCulture;

        int codigo, horasTrabalhadas;
        double valorPorHora, salario;

        codigo = int.Parse(Console.ReadLine());
        horasTrabalhadas = int.Parse(Console.ReadLine());
        valorPorHora = double.Parse(Console.ReadLine(), CI);

        salario = horasTrabalhadas * valorPorHora;

        Console.WriteLine("NUMBER = " + codigo);
        Console.WriteLine("SALARY = U$ " + salario.ToString("F2", CI));
    }
}

File: cpp/beecrowd1008.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {

    int codigo, horasTrabalhadas;
    double valorPorHora, salario;

    cin >> codigo >> horasTrabalhadas >> valorPorHora;

    salario = horasTrabalhadas * valorPorHora;

    cout << fixed << setprecision(2);
    cout << "NUMBER = " << codigo << endl;
    cout << "SALARY = U$ " << salario << endl;

    return 0;
}
